wiki/Tournaments/CWC/2023/en.md

```diff
@@ -115,6 +115,7 @@ The complete sign-up list can be found [here](https://gist.github.com/LeoFLT/1c2
 | South Korea ::{ flag=KR }:: | ::{ flag=NO }:: Norway | [May 27 (Sat) 11:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20230527T110000&p1=1440&p2=235&p3=187) |
 | China ::{ flag=CN }:: | ::{ flag=SE }:: Sweden | [May 27 (Sat) 13:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20230527T130000&p1=1440&p2=33&p3=239) |
 | Spain ::{ flag=ES }:: | ::{ flag=VN }:: Vietnam | [May 27 (Sat) 15:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20230527T150000&p1=1440&p2=141&p3=95) |
+| Red Team | Blue Team | [May 27 (Sat) 16:30 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20230527T163000&p1=1440) |
```

Comment:
i may be wrong but don't these usually have a "showmatch" footnote?

Reply:
Correct, I forgot to add those since the footnotes don't start appearing until Quarterfinals